### DISCUSSION TITLE
Issues with android app

### DISCUSSION THREAD

**Discussion by Andy3dprinter** [community/community#67643] [Mobile]
I'm not sure where to post this, but I get this error message when I try to open any repositories via the android app. Is it something I am doing wrong? And I just noticed when I click on anything now I am also getting this error. So, for example, I wanted to open the bigtreetech pad 7 section, and I cannot get in. I can search for them. However, as soon as I click on it. The same type of error messages comes up

 ![Screenshot_20230919_183946_GitHub.jpg](https://github.com/community/community/assets/127630891/342a94a6-543a-44ca-8d38-2951149e73e0)

**Comment by stevepopovich** [comment]
Hi @Andy3dprinter, could you try reinstalling the app please? If that doesn't help, we can dive deeper for sure.

**Reply by Andy3dprinter** [reply]
OK. I will try that. Thanks
